### PR TITLE
lizard -> unathi

### PR DESCRIPTION
**Pull request by FlattestGuitar**
The derogatory use of "lizard" as an insult towards Unathi stays in quite a few places.
Otherwise the species is now referred to as "Unathi".

👌 

Surprisingly, in /tg/ code there is not a single occurence of the word "ligger".

### CODE REVIEW

**Comment by Birdtalon** [comment]
Travis

**Comment by AndrewMontagne** [comment]
[Rename Lizardmen something else. Potentially Unathi](https://trello.com/c/rXfr1Eqt/45-rename-lizardmen-something-else-potentially-unathi)

File: code/game/machinery/limbgrower.dm [hunk 1]
```diff
@@ -27,7 +27,8 @@
 	var/screen = 1
 	var/list/categories = list(
 							"human",
-							"lizard",
+							"unathi",
+							"ethari",
```
Comment (SamHPurp) [inline code/game/machinery/limbgrower.dm]: 🤦‍♂️

File: code/game/machinery/limbgrower.dm [hunk 2]
```diff
@@ -136,7 +137,7 @@
 	//i need to create a body part manually using a set icon (otherwise it doesnt appear)
 	var/obj/item/bodypart/limb
 	limb = new buildpath(loc)
-	if(selected_category=="human" || selected_category=="lizard") // Doing this because plasmamen have their limbs in a different icon file
+	if(selected_category=="human" || selected_category=="unathi" || selected_category == "ethari") // Doing this because plasmamen have their limbs in a different icon file
```
Comment (SamHPurp) [inline code/game/machinery/limbgrower.dm]: Can you please fix the argument spacing

File: code/modules/clothing/head/misc.dm [hunk 3]
```diff
@@ -258,8 +258,8 @@
 	desc = "Welcome to the rice fields, motherfucker."
 	icon_state = "rice_hat"
 
-/obj/item/clothing/head/lizard
-	name = "lizardskin cloche hat"
+/obj/item/clothing/head/unathi
+	name = "unathi skin cloche hat"
 	desc = "How many lizards died to make this hat? Not enough."
```
Comment (SamHPurp) [inline code/modules/clothing/head/misc.dm]: unathi

File: code/modules/events/sentience.dm [hunk 4]
```diff
@@ -15,7 +15,7 @@
 
 	var/data = pick("scans from our long-range sensors", "our sophisticated probabilistic models", "our omnipotence", "the communications traffic on your station", "energy emissions we detected", "\[REDACTED\]")
 	var/pets = pick("animals/bots", "bots/animals", "pets", "simple animals", "lesser lifeforms", "\[REDACTED\]")
-	var/strength = pick("human", "moderate", "lizard", "security", "command", "clown", "low", "very low", "\[REDACTED\]")
+	var/strength = pick("human", "moderate", "Unathi", "security", "command", "clown", "low", "very low", "\[REDACTED\]")
```
Comment (SamHPurp) [inline code/modules/events/sentience.dm]: jesus

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm [hunk 5]
```diff
@@ -30,7 +30,7 @@
 	subcategory = CAT_MEAT
 
 /datum/crafting_recipe/food/tailkebab
-	name = "Lizard tail kebab"
+	name = "unathi tail kebab"
```
Comment (SamHPurp) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm]: Fix capitalisation please

File: code/modules/research/designs/limbgrower_designs.dm [hunk 6]
```diff
@@ -8,31 +8,31 @@
 	build_type = LIMBGROWER
 	reagents_list = list("synthflesh" = 25)
 	build_path = /obj/item/bodypart/l_arm
-	category = list("initial","human","lizard","plasmaman")
+	category = list("initial","human","lizard","plasmaman", "ethari")
```
Comment (SamHPurp) [inline code/modules/research/designs/limbgrower_designs.dm]: Please make the argument spacing consistent.

File: code/modules/research/designs/limbgrower_designs.dm [hunk 7]
```diff
@@ -8,31 +8,31 @@
...
 
 /datum/design/rightarm
 	name = "Right Arm"
 	id = "rightarm"
 	build_type = LIMBGROWER
 	reagents_list = list("synthflesh" = 25)
 	build_path = /obj/item/bodypart/r_arm
-	category = list("initial","human","lizard","plasmaman")
+	category = list("initial","human","lizard","plasmaman", "ethari")
```
Comment (SamHPurp) [inline code/modules/research/designs/limbgrower_designs.dm]: Please make the argument spacing consistent.

File: code/modules/research/designs/limbgrower_designs.dm [hunk 8]
```diff
@@ -8,31 +8,31 @@
...
 
 /datum/design/leftleg
 	name = "Left Leg"
 	id = "leftleg"
 	build_type = LIMBGROWER
 	reagents_list = list("synthflesh" = 25)
 	build_path = /obj/item/bodypart/l_leg
-	category = list("initial","human","lizard","plasmaman")
+	category = list("initial","human","lizard","plasmaman", "ethari")
```
Comment (SamHPurp) [inline code/modules/research/designs/limbgrower_designs.dm]: Please make the argument spacing consistent.

File: code/modules/research/designs/limbgrower_designs.dm [hunk 9]
```diff
@@ -8,31 +8,31 @@
...
 
 /datum/design/rightleg
 	name = "Right Leg"
 	id = "rightleg"
 	build_type = LIMBGROWER
 	reagents_list = list("synthflesh" = 25)
 	build_path = /obj/item/bodypart/r_leg
-	category = list("initial","human","lizard","plasmaman")
+	category = list("initial","human","lizard","plasmaman", "ethari")
```
Comment (SamHPurp) [inline code/modules/research/designs/limbgrower_designs.dm]: Please make the argument spacing consistent.

Comment (SamHPurp) [reply code/modules/research/designs/limbgrower_designs.dm]: Also while these appear to be Ethari bug fixes, shouldn't the lizard's be unathi?

File: code/modules/research/designs/limbgrower_designs.dm (hunk 8 above)
Comment (SamHPurp) [reply code/modules/research/designs/limbgrower_designs.dm]: Also while these appear to be Ethari bug fixes, shouldn't the lizard's be unathi?

File: code/modules/research/designs/limbgrower_designs.dm (hunk 7 above)
Comment (SamHPurp) [reply code/modules/research/designs/limbgrower_designs.dm]: Also while these appear to be Ethari bug fixes, shouldn't the lizard's be unathi?

File: code/modules/research/designs/limbgrower_designs.dm (hunk 6 above)
Comment (SamHPurp) [reply code/modules/research/designs/limbgrower_designs.dm]: Also while these appear to be Ethari bug fixes, shouldn't the lizard's be unathi?

**Comment by SamHPurp** [comment]
You missed the /s off of your commit name

**Pull request merged**